synapse/models/infotech.py

```diff
@@ -301,6 +301,16 @@ def getModelDefs(self):
                 ('it:app:snort:hit', ('guid', {}), {
                     'doc': 'An instance of a snort rule hit.',
                 }),
+                ('it:reveng:function', ('guid', {}), {
+                    'doc': 'A function inside an executable',
+                }),
+                ('it:reveng:function:string', ('comp', {'fields': (('function', 'it:reveng:function'), ('string', 'str'))}), {
```

Comment:
can you place the form declaration of `it:reveng:file:function` before `it:reveng:function:string` ? keeps the additions in order.

Reply:
:+1: